sbnana/CAFAna/XSec/Flux.h

```diff
@@ -24,7 +24,19 @@ namespace ana
                 Color_t col = kBlack,
                 Style_t style = kSolid,
                 EBinType bintype = kBinContent);
+
+    void SaveTo(TDirectory* dir, const std::string& name) const;
+
+    static std::unique_ptr<FluxTimesNuclei> LoadFrom(TDirectory* dir, const std::string& name);
+
+    /// Convert an \ref FluxTimesNuclei into a \ref Spectrum where every
+    /// bin is the integral of the \ref FluxTimesNuclei
```

Comment:
Can you add a few more words about the use of this in non-differential xsec measurements?